subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -4494,35 +4521,30 @@ static inline int length_req_rsp_recv(struct ll_conn *conn, memq_link_t *link,
 		u16_t max_rx_time;
 		u16_t max_tx_time;
 
+
+		dle_max_time_get(conn, &max_rx_time, &max_tx_time);
+
 		/* use the minimal of our default_tx_time and
 		 * peer max_rx_time
 		 */
-		max_rx_time = sys_le16_to_cpu(lr->max_rx_time);
-		if (max_rx_time >= PKT_US(PDU_DC_PAYLOAD_SIZE_MIN, 0)) {
-			eff_tx_time = MIN(max_rx_time,
-					  conn->default_tx_time);
+		if (lr->max_rx_time >= PKT_US(PDU_DC_PAYLOAD_SIZE_MIN, 0)) {
```

Comment:
put back the `sys_le16_to_cpu`

Reply:
Yes, these conversions must be applied to references to lr->max_xxxxx. Should probably be done by assigning a local variable

Reply:
introduced local vars